healthcheck_schema.py

```diff
@@ -0,0 +1,22 @@
+import graphene
+
+from graphene_sqlalchemy import (SQLAlchemyObjectType)
+
+from api.room.models import Room as RoomModel
+
+
+class Room(SQLAlchemyObjectType):
+    class Meta:
+        model = RoomModel
+
+
+class Query(graphene.ObjectType):
+    rooms = graphene.List(Room)
+
+    def resolve_rooms(self, info):
+        query = Room.get_query(info)
+        res = query.all()
+        return res
+
+
+healthcheck_schema = graphene.Schema(query=Query)
```

Comment:
I think what should be in this module must look like what is in the **main.py** module.

Reply:
@PatrickCmd ,I dont get what you saying.Please rephrase your feedback.